<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompetitorInspector</class>
 <widget class="QDockWidget" name="CompetitorInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Competitor Inspector</string>
  </property>
  <widget class="QWidget" name="m_widget_container">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QToolBar" name="m_tool_bar">
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <addaction name="m_action_add"/>
      <addaction name="m_action_remove"/>
      <addaction name="m_action_edit"/>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="m_table_view"/>
    </item>
   </layout>
  </widget>
  <action name="m_action_add">
   <property name="icon">
    <iconset resource="../../../common/icons/chrono.qrc">
     <normaloff>:/common/edit_add.png</normaloff>:/common/edit_add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="m_action_remove">
   <property name="icon">
    <iconset resource="../../../common/icons/chrono.qrc">
     <normaloff>:/common/edit_remove.png</normaloff>:/common/edit_remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="m_action_edit">
   <property name="icon">
    <iconset resource="../../../common/icons/chrono.qrc">
     <normaloff>:/common/edit.png</normaloff>:/common/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../common/icons/chrono.qrc"/>
 </resources>
 <connections/>
</ui>
